### DISCUSSION TITLE
0410. 分割数组的最大值

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#150] [Comments]
# 0410. 分割数组的最大值

[https://algo.itcharge.cn/solutions/0400-0499/split-array-largest-sum/](https://algo.itcharge.cn/solutions/0400-0499/split-array-largest-sum/)

**Comment by T1anq1** [comment]
本题翻译出现了瑕疵，"给定一个非负整数数组 nums 和一个整数 k，将数组分成 m 个非空的连续子数组",应当改为将数组分成k个非空的连续子数组。后续的所有m都应该同样改为k, 不然会造成很大的歧义。

**Reply by itcharge** [reply]
感谢指正，文中已更新
